src/components/Terminal/index.tsx

```diff
@@ -85,6 +86,10 @@ class Terminal extends React.Component<ThemedTerminalProps, {}> {
 			}
 			window.addEventListener('resize', this.resize.bind(this));
```

Comment:
Why do we need both this and a resize observer?

Reply:
we don't, I was sure I removed it... thanks for noticing